test_scripts/Policies/Policy_Table_Update/177_PTU_is_performed_for_several_Apps_if_HMI_sends_BC.OnSystemRequest_with_url_parameter.lua

```diff
@@ -0,0 +1,80 @@
+---------------------------------------------------------------------------------------------------
+-- Issues: https://github.com/smartdevicelink/sdl_core/issues/3715,
+--         https://github.com/smartdevicelink/sdl_core/issues/3881
+---------------------------------------------------------------------------------------------------
+-- Description:
+--  SDL accept PTU from application for several Apps in case HMI sends BC.OnSystemRequest with 'url' parameter
+-- Steps:
+-- 1. Core and HMI are started
+-- 2. App1 is registered
+-- 3. App2 is registered
+-- 3. Core sends OnStatusUpdate UPDATING
```

Comment:
```suggestion
-- 4. Core sends OnStatusUpdate UPDATING
```
and please update other numbers below